database/console/show_command.go

```diff
@@ -73,82 +71,50 @@ func (r *ShowCommand) Handle(ctx console.Context) error {
 		return nil
 	}
+
 	r.schema = r.schema.Connection(ctx.Option("database"))
-	connection := r.schema.GetConnection()
-	getConfigValue := func(k string) string {
-		return r.config.GetString("database.connections." + connection + "." + k)
-	}
+	dbConfig := r.schema.Orm().Config()
 	info := databaseInfo{
-		Database: getConfigValue("database"),
-		Host:     getConfigValue("host"),
-		Port:     getConfigValue("port"),
-		Username: getConfigValue("username"),
+		Database: dbConfig.Database,
+		Host:     dbConfig.Host,
+		Port:     dbConfig.Port,
+		Username: dbConfig.Username,
 	}
+
 	var err error
 	info.Name, info.Version, info.OpenConnections, err = r.getDataBaseInfo()
 	if err != nil {
 		ctx.Error(err.Error())
 		return nil
 	}
+
 	if info.Tables, err = r.schema.GetTables(); err != nil {
 		ctx.Error(err.Error())
 		return nil
 	}
+
 	if ctx.OptionBool("views") {
 		if info.Views, err = r.schema.GetViews(); err != nil {
 			ctx.Error(err.Error())
 			return nil
 		}
 	}
 	r.display(ctx, info)
+
 	return nil
 }
 
-func (r *ShowCommand) getDataBaseInfo() (name, version, openConnections string, err error) {
-	var (
-		drivers = map[string]struct {
-			name                 string
-			versionQuery         string
-			openConnectionsQuery string
-		}{
-			database.DriverSqlite.String(): {
-				name:         "SQLite",
-				versionQuery: "SELECT sqlite_version() AS value;",
-			},
-			database.DriverMysql.String(): {
-				name:                 "MySQL",
-				versionQuery:         "SELECT VERSION() AS value;",
-				openConnectionsQuery: "SHOW status WHERE variable_name = 'threads_connected';",
-			},
-			database.DriverPostgres.String(): {
-				name:                 "PostgresSQL",
-				versionQuery:         "SELECT current_setting('server_version') AS value;",
-				openConnectionsQuery: "SELECT COUNT(*) AS value FROM pg_stat_activity;",
-			},
-			database.DriverSqlserver.String(): {
-				name:                 "SQL Server",
-				versionQuery:         "SELECT SERVERPROPERTY('productversion') AS value;",
-				openConnectionsQuery: "SELECT COUNT(*) Value FROM sys.dm_exec_sessions WHERE status = 'running';",
-			},
-		}
-	)
-	name = r.schema.Orm().Query().Driver()
-	if driver, ok := drivers[name]; ok {
-		var versionResult queryResult
-		if err = r.schema.Orm().Query().Raw(driver.versionQuery).Scan(&versionResult); err == nil {
-			version = versionResult.Value
-			if strings.Contains(version, "MariaDB") {
-				name = "MariaDB"
-				version = strings.Trim(version[:strings.Index(version, "MariaDB")], "-")
-			}
-			if len(driver.openConnectionsQuery) > 0 {
-				var openConnectionsResult queryResult
-				if err = r.schema.Orm().Query().Raw(driver.openConnectionsQuery).Scan(&openConnectionsResult); err == nil {
-					openConnections = openConnectionsResult.Value
-				}
-			}
-		}
+func (r *ShowCommand) getDataBaseInfo() (name, version string, openConnections int, err error) {
+	name = r.schema.Orm().Name()
```

Comment:
The database names should be PostgreSQL, MySQL, MariaDB, SQL Server, and SQLite, not lowercase driver name like postgres, mysql, sqlite, or sqlserver